independent-projects/vertx-java-io/src/main/java/io/quarkus/vertx/java/io/VertxBufferImpl.java

```diff
@@ -9,10 +9,14 @@
 import io.netty.buffer.Unpooled;
 import io.netty.util.CharsetUtil;
 import io.vertx.core.buffer.Buffer;
+import io.vertx.core.buffer.impl.BufferImpl;
 import io.vertx.core.impl.Arguments;
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
 
+/**
+ * A variant of {@link BufferImpl} doing no bound checks for performance reasons.
+ */
 public class VertxBufferImpl implements Buffer {
```

Comment:
Maybe we could also try to use a better name since it would be now part of the public API...

Reply:
Proposals?

Reply:
Yes, that's a little bit harder. `NoBoundChecksBufferImpl`? Maybe `QuarkusBufferImpl` - I saw this naming strategy here and there...

Reply:
`QuarkusBufferImpl` would be a good name, but this is a module in independent-projects :)

I would simply go with `NoBoundChecksBuffer`

Reply:
Thanks for the feedback, let me rename it to `NoBoundChecksBuffer` then.